### DISCUSSION TITLE
Welcome to Asphalt9Materials Discussions!

### DISCUSSION THREAD

**Discussion by c-Cyril-l** [c-Cyril-l/Asphalt9Materials#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JonChoice** [comment]
Hi, is there a way to keep a log of what time I do on each track? a way to populate a database that I could query such as:
What is my best time on this track with any car
What is my best time on this track with a C class car

If I could query the Asphalt db directly with my user ID it would be great or I could maybe run a listener on my network whilst I play on the same wifi and sniff what data is sent (if not encrypted).

anyways in the work you have done would you say such is possible ?

**Comment by ellmoe2000** [comment]
I came upon this today. I am an avid player and thought this project would be beneficial to me as to how I could better progress through the game. 

I tried installing like your video teaches, but couldn’t download the sql server. Could I get some help solving this issue. 

Thank you
